src/WebJobs.Script/Workers/Rpc/Configuration/RpcWorkerConfigFactory.cs

```diff
@@ -159,6 +168,59 @@ internal void AddProvider(string workerDir)
             }
         }
 
+        private List<WorkerDescriptionProfile> ReadWorkerDescriptionFromProfiles(JObject workerConfig)
+        {
+            List<WorkerDescriptionProfile> descriptionProfiles = new List<WorkerDescriptionProfile>();
```

Comment:
nit: this can just be var since it's very clear what the type is on the right side of the expression
```suggestion
            var descriptionProfiles = new List<WorkerDescriptionProfile>();
```